src/main/scala/scalismo/common/DiscreteField.scala

```diff
@@ -78,6 +81,15 @@ class DiscreteVectorField[D <: Dim, DO <: Dim] private (val domain: DiscreteDoma
   override def apply(ptId: Int) = data(ptId)
   override def isDefinedAt(ptId: Int) = data.isDefinedAt(ptId)
 
+  
+  /**
+   * Returns a continuous vector field, where the value at each point is that of the closest point in the discrete set 
+   * **/
+  def interpolateNearestNeighbor()(implicit ev: NDSpace[D], cb: CanBound[D]): VectorField[D, DO] = {
+    val indexedDomain = SpatiallyIndexedDiscreteDomain(domain.points.toIndexedSeq, domain.numberOfPoints)(ev)
```

Comment:
For computational reasons, we might want to do this only if the domain is not yet SpatiallyIndexed (we can find this out by pattern matching). This would make it possible to apply it also to e.g. Triangle Meshes, without incurring twice the costs of indexing